.config/1espt/PipelineAutobaseliningConfig.yml

```diff
@@ -0,0 +1,12 @@
+## DO NOT MODIFY THIS FILE MANUALLY. This is part of auto-baselining from 1ES Pipeline Templates. Go to [https://aka.ms/1espt-autobaselining] for more details.
+
+pipelines:
+  3057:
```

Comment:
@weshaggard I wonder if they are just doing this for template since it's the only pipeline marked for canary, and eventually ALL pipelines will show up this config. I'll follow up.

Reply:
Not sure we should figure out the testing. I'm also not very happy with how noisy this is as it seems like it is going to create a PR every time we build which isn't ideal.